Automattic/wp-calypso
# Deduplicate regenerate-unicode-properties

#### Changes proposed in this Pull Request

* Deduplicate `regenerate-unicode-properties` (done automatically with `npx yarn-deduplicate --packages regenerate-unicode-properties`)

#### Testing instructions

* Verify tests still pass

For reference, these are the top-level dependencies affected by this change

```diff
< Before
> After
---
< wp-calypso-monorepo@0.17.0 -> @automattic/calypso-build@6.1.0 -> ... -> regenerate-unicode-properties@8.1.0
< wp-calypso-monorepo@0.17.0 -> @storybook/react@5.3.18 -> ... -> regenerate-unicode-properties@8.1.0
< wp-calypso-monorepo@0.17.0 -> @wordpress/scripts@7.2.0 -> ... -> regenerate-unicode-properties@8.1.0
< wp-calypso-monorepo@0.17.0 -> calypso-codemods@0.1.6 -> ... -> regenerate-unicode-properties@8.1.0
< wp-calypso-monorepo@0.17.0 -> wp-calypso@0.17.0 -> ... -> regenerate-unicode-properties@8.1.0
< wp-calypso-monorepo@0.17.0 -> wp-e2e-tests@0.0.1 -> ... -> regenerate-unicode-properties@8.1.0
> wp-calypso-monorepo@0.17.0 -> @automattic/calypso-build@6.1.0 -> ... -> regenerate-unicode-properties@8.2.0
> wp-calypso-monorepo@0.17.0 -> @storybook/react@5.3.18 -> ... -> regenerate-unicode-properties@8.2.0
> wp-calypso-monorepo@0.17.0 -> @wordpress/scripts@7.2.0 -> ... -> regenerate-unicode-properties@8.2.0
> wp-calypso-monorepo@0.17.0 -> calypso-codemods@0.1.6 -> ... -> regenerate-unicode-properties@8.2.0
> wp-calypso-monorepo@0.17.0 -> wp-calypso@0.17.0 -> ... -> regenerate-unicode-properties@8.2.0
> wp-calypso-monorepo@0.17.0 -> wp-e2e-tests@0.0.1 -> ... -> regenerate-unicode-properties@8.2.0
```
